suave/sol/libraries/Suave.sol

```diff
@@ -105,6 +107,15 @@ library Suave {
         }
     }
 
+    function ethcall(address contractAddr, bytes memory input1) internal view returns (bytes memory) {
```

Comment:
Hm, how would we handle errors? Do we expect to return the encoded error in bytes here and is that the case?